README.md

```diff
@@ -93,7 +93,7 @@ Then elsewhere in your code:
 var Plotly = require('./path/to/custom-plotly');
 ```
 
-To learn more about the plotly.js module architecture, refer to our [modularizing monolithic JS projects](https://plot.ly/javascript/modularizing-monolithic-javascript-projects/) post.
+To learn more about the plotly.js module architecture, refer to our [modularizing monolithic JS projects](https://plotly.com/javascript/plotly-fundamentals/) post.
```

Comment:
Let's just remove this line please. The underlying content has been removed.

Reply:
Done in https://github.com/plotly/plotly.js/pull/4694/commits/1fe9c53d66973721d0a512c4570b9c7d612b1f90.